application/src/main/resources/thingsboard.yml

```diff
@@ -14,12 +14,22 @@
 # limitations under the License.
 #
 
+# Thingsboard upgrade parameters
+upgrade:
+  # Enable/disable ThingsBoard upgrade
+  do_upgrade: "${UPGRADE_THINGSBOARD:false}"
+# ThingsBoard installation parameters
+install:
+  # Enable/disable ThingsBoard installation
+  upgrade: "${INSTALL_UPGRADE:false}"
+  # ThingsBoard installation directory
+  data_dir: "${INSTALL_DATA_DIRECTORY:}"
```

Comment:
Bunu eklememiz gerekiyor mu? Default'ta bu yokken bizim eklemize neden ihtiyaç oldu?

Reply:
Tüm install grubu için

Reply:
Bu orjinalinde varken kaldırılmıştı o yüzden kendimiz ekledik. Sonraki versiyonda orjinaline de eklemişler tekrardan